### PR TITLE
[cluster-test] Allow to customize bench experiment duration

### PR DESCRIPTION
**Pull request by andll**
For cluster test pre-land we can reduce it from 5 minutes to 2 by using `--duration 60` (2 min total = 1 min duration + 1 min buffer)

### CODE REVIEW

File: testsuite/cluster-test/src/experiments/performance_benchmark_nodes_down.rs [hunk 1]
```diff
@@ -82,12 +103,12 @@ impl Experiment for PerformanceBenchmarkNodesDown {
             .collect();
         let futures = stop_effects.iter().map(|e| e.activate());
         join_all(futures).await;
-        let window = Duration::from_secs(240);
+        let buffer = Duration::from_secs(30);
```
Comment (sausagee) [inline testsuite/cluster-test/src/experiments/performance_benchmark_nodes_down.rs]: heh, simple LPF :)

File: testsuite/cluster-test/src/experiments/performance_benchmark_nodes_down.rs [hunk 2]
```diff
@@ -36,12 +36,31 @@ pub struct PerformanceBenchmarkNodesDownParams {
         help = "Whether cluster test should run against validators or full nodes"
     )]
     pub is_fullnode: bool,
+    #[structopt(
+    long,
+    default_value = stringify!(DEFAULT_BENCH_DURATION),
+    help = "Number of nodes which should be down"
```
Comment (ankushagarwal) [inline testsuite/cluster-test/src/experiments/performance_benchmark_nodes_down.rs]: nit: This description should be updated

File: scripts/cti [hunk 3]
```diff
@@ -58,6 +63,7 @@ then
       echo "-W|--workspace <WORKSPACE>: Use custom workplace <WORKSPACE>"
       echo "-E|--env <ENV>: Set environment variable, ex. -E RUST_LOG=debug. Can be repeated, e.g. -E A=B -E C=D"
       echo "-R|--report file.json: Generate json report into file.json"
+      echo "--marker <MARKER>: Marker to identify purpose of this run. Must be set by automatic invocations like CI"
```
Comment (sausagee) [inline scripts/cti]: Curious, what does it actually do behind the scene in CTI?

**Comment by ankushagarwal** [comment]
We should at least run this for `--duration 120`. In a 60 second period, the validator only produces 4 data points (at a 15 sec collection rate) - I think this could introduce variance in the results.

File: scripts/cti (hunk 3 above)
Comment (andll) [reply scripts/cti]: For now it just logs with this marker, so we can understand what is going on

**Pull request closed without merging**